react/features/face-landmarks/FaceLandmarksHelper.ts

```diff
@@ -175,6 +199,26 @@ export class HumanHelper implements FaceLandmarksHelper {
             faceExpression = this.getFaceExpression({ detections });
         }
 
+        if (this.faceDetectionTypes.includes(DETECTION_TYPES.AGE)) {
+            if (!detections) {
```

Comment:
this 3 line chunk seems to be repeated a few times, doesn't it make sense to check for that first?